pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/sort/Alias.java

```diff
@@ -86,8 +89,10 @@ public static Alias of(SortBy sortBy) {
 
     public static Alias of(ParticipantColumn column) {
         Alias esAlias;
-        if (Objects.nonNull(column.getObject())) {
+        if (Objects.nonNull(column.getObject()) && Alias.ofOrNull(column.getObject()) != null) {
             esAlias = Alias.of(column.getObject());
+        } else if (ElasticSearchUtil.QUESTIONS_ANSWER.equals(column.getObject())) {
```

Comment:
is this else if really necessary? else block calls `Alias.of` which automatically returns `ACTIVITIES` alias if there is no such enum

Reply:
I think this was necessary to guard against the edge case where a tableAlias might be duplicative of an activity name.

Reply:
if something is passed to this method which does not present in `Alias` enum class it will return `ACTIVITIES` by default and `QUESTIONS_ANSWER` does not exist as enum, that's why I got this question